File: SimpleLinearRegression/Part 3 - Regression table/simple_linear_regression_table.py
import pandas as pd
import statsmodels.api as sm

# Read the CSV data
DF = pd.read_csv("example_data_simple_linear_regression.csv")

# Fit a linear model
LM = sm.OLS(DF['height'], sm.add_constant(DF['weight'])).fit()

# Print a regression table
print(LM.summary2())

File: SimpleLinearRegression/Part 3 - Regression table/simple_linear_regression_table.R
# Store this script as well as the data set in the same location, then go to:
# Session > Set Working Directory > To Source File Location
DF <- read.csv("example_data_simple_linear_regression.csv")

# Fit a linear model
LM <- lm(height ~ weight, data = DF)

# Print a regression table
summary(LM)